Dockerfile.alpine.template

```diff
@@ -0,0 +1,58 @@
+FROM postgres:%%PG_MAJOR%%-alpine
+MAINTAINER Régis Belson <me@regisbelson.fr>
+
+ENV POSTGIS_VERSION %%POSTGIS_VERSION%%
+ENV POSTGIS_SHA256 %%POSTGIS_SHA256%%
+
+RUN set -ex \
+    \
+    && apk add --no-cache --virtual .fetch-deps \
+        ca-certificates \
+        openssl \
+        tar \
+    \
+    && wget -O postgis.tar.gz "https://github.com/postgis/postgis/archive/$POSTGIS_VERSION.tar.gz" \
+    && echo "$POSTGIS_SHA256 *postgis.tar.gz" | sha256sum -c - \
+    && mkdir -p /usr/src/postgis \
+    && tar \
+        --extract \
+        --file postgis.tar.gz \
+        --directory /usr/src/postgis \
+        --strip-components 1 \
+    && rm postgis.tar.gz \
+    \
+    && apk add --no-cache --virtual .build-deps \
+        autoconf \
+        automake \
+        g++ \
+        json-c-dev \
+        libtool \
+        libxml2-dev \
+        make \
+        perl \
+    \
+    && apk add --no-cache --virtual .build-deps-testing \
```

Comment:
I had to do it like this instead of just using tagged repositories because of [this bug](https://github.com/gliderlabs/docker-alpine/issues/205).